src/commands/hosting-channel-deploy.ts

```diff
@@ -28,9 +28,18 @@ interface ChannelInfo {
   target: string | null;
   site: string;
   url: string;
+  version: string;
   expireTime: string;
 }
 
+interface DeployResult {
+  hosting: {
+    [site: string]: {
+      [key: string]: string;
+    };
```

Comment:
Might the value of this map be the Release interface in `hosting/api.ts`? (I don't know off hand, but it would make for stronger typing)

Reply:
This is a great idea. It would require a bit more code changes to capture the payload returned by the hosting release api, but it makes more sense to me. Let me give it a try.